server/controllers/tripController.js

```diff
@@ -50,6 +61,41 @@ class TripController {
       data: saveTrip
     });
   }
+
+  /**
+   * This method handle oneWayTrip request.
```

Comment:
Hi @KKDoddy , "This method handle oneWayTrip request.", but I see this "tripType, from, to, date, reasons, accommodationId, returnDate"